low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -193,6 +195,13 @@ def initialize(self, conf_reader, msgQlist, product):
         self._drive_byid_by_serial_number = {}
 
         self._import_products(product)
+        self.utility = Utility()
+
+        # Flags describing ping status to the other node over pvt data nw
+        self.prev_is_other_node_reachable = False

```

Comment:
think of this how it will work in LDR R2 where more than 2 nodes would be there. Cortx may or may not this functionality in R2 but code written now can be generic enough.

keep a dict with interface names as keys & their connectivity (reachable) status as true or false. in LDR R1 this array can have only one entry for other node.